src/content/tools/mcp-server.md

```diff
@@ -27,24 +27,30 @@ The Dart MCP server provides a growing list of tools that
 grant AI assistants deep insights into your project.
 Here is an overview of a few things it can do:
 
-*  Analyze and fix errors in your project's code.
-*  Introspect and interact with your running application
-   (such as trigger a hot reload, get the selected widget,
-   fetch runtime errors).
+*  Analyze and fix diagnostics in your project's code.
+*  Resolve symbols to elements to ensure their existence and
+   fetch documentation and signature information for them.
+*  Introspect and interact with your running application,
+   such as fetching runtime errors.
+   For Flutter apps, it can trigger a hot reload, get a selected widget,
+   inspect the widget tree, take screenshots, and more.
```

Comment:
```suggestion
*  Introspect and interact with your running application.
```

I just realized we snuck some things about Flutter into the Dart docs here. Let's strip as much as we can or minimize. Perhpase we can provide how-tos for Flutter-specific tasks in the Flutter documentation.